retrofit/src/test/java/retrofit/RequestBuilderTest.java

```diff
@@ -910,6 +910,21 @@
     assertThat(request.body()).isNull();
   }
 
+  @Test public void getWithUrlAbsoluteDifferentHost() {
```

Comment:
That's fine, didn't see the .org difference. Any value in a test with the same authority?